xbmc/addons/GUIDialogAddonInfo.cpp

```diff
@@ -197,8 +198,57 @@ void CGUIDialogAddonInfo::UpdateControls()
 
 void CGUIDialogAddonInfo::OnUpdate()
 {
-  CAddonInstaller::GetInstance().InstallOrUpdate(m_addon->ID());
   Close();
+
+  if (!m_localAddon)
+    return;
+
+  auto* dialog = static_cast<CGUIDialogSelect*>(g_windowManager.GetWindow(WINDOW_DIALOG_SELECT));
```

Comment:
It's referenced before that, but I'm assuming you meant 'move it a bit further down'